erp/management/commands/scrapfly_acquisition.py

```diff
@@ -0,0 +1,266 @@
+import copy
+import json
+import math
+import re
+from collections import defaultdict
+from urllib.parse import urlencode
+from uuid import uuid4
+
+import reversion
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from rest_framework.exceptions import ValidationError
+from scrapfly import ScrapeConfig, ScrapflyClient, ScrapflyScrapeError
+
+from erp.imports.serializers import ErpImportSerializer
+from erp.models import Erp
+from erp.provider import geocoder
+
+# NOTE: mainly based on https://github.com/scrapfly/scrapfly-scrapers/blob/main/bookingcom-scraper/bookingcom.py
+
+
+class Command(BaseCommand):
+    help = "Create ERPs using scrapfly"
+
+    headers = {
+        "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/62.0.3202.94 Safari/537.36",
+        "Accept-Encoding": "gzip, deflate, br",
+        "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8",
+        "Connection": "keep-alive",
+        "Accept-Language": "fr-FR",
+    }
+
+    base_config = {
+        # Booking.com requires Anti Scraping Protection bypass feature:
+        "asp": True,
+        "country": "FR",
+    }
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--query",
+            type=str,
+            required=True,
+            help="The query you want to search on Outscraper to insert new ERPs",
+        )
+
+    def _convert_access(self, access):
+        accessibility = {}
+        access_copy = copy.deepcopy(access)
+
+        if (key := "Toilettes avec barres d'appui") in access_copy:
+            accessibility["sanitaires_presence"] = True
+            access_copy.remove(key)
+
+        if (key := "Accessible en fauteuil roulant") in access_copy:
+            accessibility["entree_plain_pied"] = True
+            accessibility["entree_largeur_mini"] = 80
+            access_copy.remove(key)
+
+        if (key := "Logement entièrement accessible en fauteuil roulant") in access_copy:
+            accessibility["accueil_chambre_nombre_accessibles"] = 1
+            access_copy.remove(key)
+
+        if (key := "Guidage vocal") in access_copy:
+            accessibility["entree_balise_sonore"] = True
+            access_copy.remove(key)
+
+        for key_to_ignore in (
+            "Logement entièrement situé au rez-de-chaussée",
+            "Cordon d'alarme dans la salle de bains",
+            "Lavabo bas adapté aux personnes à mobilité réduite",
+            "Toilettes surélevées",
+            "Aides visuelles : braille",
+            "Étages supérieurs accessibles par ascenseur",
+            "Étages supérieurs accessibles uniquement par les escaliers",
+            "Aides visuelles : panneaux tactiles",
+        ):
+            if key_to_ignore not in access_copy:
+                continue
+            access_copy.remove(key_to_ignore)
```

Comment:
```suggestion
            access_copy.pop(key_to_ignore, None)
```
I think this could work and would be simpler, what do you think ?

Reply:
access is a list, not a dict